build.gradle.kts

```diff
@@ -66,7 +66,8 @@ java {
 }
 
 apiValidation {
-    ignoredPackages.add("com.chattriggers.ctjs.internal")
+    ignoredProjects += "typing-generator"
+    ignoredPackages += "com.chattriggers.ctjs.internal"
```

Comment:
Does this remove our dependency on EssentialGG/essential-gradle-toolkit#15?

Reply:
We already migrated away from egt in a previous PR.

Reply:
so true